### PR TITLE
doc(changelogs): add 3.0 summary

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

File: docs/changes/3.0.0.rst [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+It can be painful to migrate a large code base to a major new version of a framework. Therefore, in 3.0 we went to great lengths to minimize breaking changes, although a number of methods and attributes were deprecated. That being said, everyone will likely run up against at least one or two items in the breaking changes list below. Please carefully review the list of changes and thoroughly test your apps with Falcon 3.0 before deploying to production.
+
+Leading up to this release, members of the core maintainers spent many hours (and not a few late nights and weekends) prototyping, tuning, and testing in order to uphold the high standards of correctness and reliability for which Falcon is known. That being said, no code is perfect, so please don't hesitate to reach out on `falconry/user
```
Comment (vytas7) [inline docs/changes/3.0.0.rst:18]: members of the core maintainers _team_ ?

**Review by vytas7 — CHANGES_REQUESTED** [review]
Awesome summary! :100: 

Those Rst lines got a bit overlong though, it would be nice to re-wrap them to 80-120 or whatever our soft limit for docs is.

File: docs/changes/3.0.0.rst (hunk 1 above)
Comment (kgriffs) [reply docs/changes/3.0.0.rst]: fixed

**Review by vytas7 — APPROVED** [review]
I've rendered a copy to do a quick sanity check on all links, and the final view... and it looks great!

:100:

**Review by CaselIT — APPROVED** [review]
lgtm

**Comment by vytas7** [comment]
Onwards! :shipit:

**Pull request merged**